benchmark/e2e/_perfshare.py

```
@@ -0,0 +1,135 @@
+
+    cat, be, total = collections.defaultdict(float), collections.defaultdict(float), 0.0
+    for ev in prof.key_averages():
+        t = ev.self_device_time_total
+        if t <= 0:
+            continue
+        cat[categorize(ev.key)] += t
+        be[backend(ev.key)] += t
+        total += t
+
+    kernel_ms = total / 1e3
+    print(f"\nfwd+bwd training step (eager):")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)